block_alerts.php

```diff
@@ -29,19 +29,18 @@ class block_alerts extends block_base {
      * @return void
      */
     public function init() {
-        global $USER;
         $this->title = get_string('pluginname', 'block_alerts');
     }
 
     /**
      * Gets the block settings.
      *
+     * @return void
      */
     public function specialization() {
-        if (isset($this->config->title)) {
```

Comment:
`$this->config->title` is set for empty string. e.g. when user add the title and then later removed it.
I think it's better to check for empty string.

Reply:
`else {
            $this->title = get_string('pluginname', 'block_alerts');
    }`
this is not necessary as it is set in the `init()` already.

Reply:
Allow for no title looks good when tested locally - thanks:)